src/app/item-page/simple/field-components/specific-field/title/item-page-title-field.component.html

```diff
@@ -1,6 +1,30 @@
-<h1 class="item-page-title-field">
+<!-- // Core code block- Left intact on purpose -->
```

Comment:
Can we please use consistency with customization comments.

Reply:
Not a TAMU customization

Reply:
<img width="656" alt="image" src="https://github.com/user-attachments/assets/dabf6052-2516-4025-b44a-80149f9d78df">

This is a core code customization removing the core template for this component and replacing it with our own.

Are we not able to override this component in the themes?